src/main/java/duke/ui/StringFormatting.java

```diff
@@ -0,0 +1,84 @@
+package duke.ui;
+
+import duke.Duke;
+import duke.io.FileManager;
+
+/**
+ * Class to help with formatting strings for printing in the UI.
+ */
+public class StringFormatting {
+
+    private static final String STRING_GREETING = "Good morning!\nWhat would you like to do today?";
+    private static final String STRING_GOODBYE = "Alright! Your list has been saved "
+            + "successfully. \n\nSee you around!";
+    private static final String STRING_LOADED = "A save file was found and loaded successfully.";
+    private static final String STRING_NOT_LOADED = "No save file was found, or the format was improper.";
+
+    /**
+     * Formats a string to say how many tasks there are based on given integer.
+     * Accounts for singular vs plural (1 task vs 2 tasks, is vs are).
+     *
+     * @param numberOfTasks number of tasks
+     * @return formatted string
+     */
+    public static String formatNumberOfTasksString(int numberOfTasks) {
+        return "\nThere " + (numberOfTasks == 1 ? "is" : "are")
+                + " currently " + numberOfTasks + " task"
+                + (numberOfTasks == 1 ? "" : "s") + " in your list.";
+    }
```

Comment:
Similar to previously, while functionally this is valid, you may want to consider splitting this logic up with the use of if-else statements, so as to make this section more reader-friendly.